fe/fe-core/src/main/java/com/starrocks/common/CaseSensibility.java

```diff
@@ -19,7 +19,7 @@
 
 public enum CaseSensibility {
     CLUSTER(true),
-    DATABASE(true),
+    DATABASE(false),
```

Comment:
After this change, the internal tables have become case-insensitive, which has a significant impact.  
It would be better if only queries to system tables were case-insensitive.  
See if there’s any other way to modify it.